libs/SalesforceSDKCore/SalesforceSDKCoreTests/SalesforceRestAPITests.m

```diff
@@ -1045,18 +1019,8 @@ - (void) testUploadShareFileSharesSharedFilesUnshareDelete {
     XCTAssertEqualObjects(listener.returnStatus, kTestRequestStatusDidLoad, @"request failed");
 }
 
-- (void)testGzip {
```

Comment:
Removed this test since it's not really adding value. It's not testing anything in our framework. It's attempting to test our service's handling of `gzip` header.